### ISSUE TITLE
[AXE-CORE]: VAMC - ARIA attributes must conform to valid values

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21786]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where ARIA attributes were not conforming to valid values. Preliminary research indicates this was mis-formed markup for an `aria-hidden="true"` attribute on right pointing icons. Links to pages below.

<details>
<summary> The violation occurs with the operating status carat </summary>

![image](https://user-images.githubusercontent.com/55411834/112065444-42c49980-8b2a-11eb-9304-0f9ee1b6a826.png)

</details

**Element location**
`.vads-u-padding-left--0p5`
**Element source**
<`i class="fa fa-chevron-right vads-u-padding-left--0p5" "="" aria-hidden=" true"></i>`

## Tasks
To solve this violation, you need to fix the following:
- Invalid ARIA attribute value: aria-hidden=" true"
- Remove the extra `"="` markup from the I tag

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Environment
* https://www.va.gov/butler-health-care/
* https://www.va.gov/butler-health-care/locations/
* https://www.va.gov/butler-health-care/locations/abie-abraham-va-clinic/
* https://www.va.gov/cheyenne-health-care/
* https://www.va.gov/cheyenne-health-care/locations/
* https://www.va.gov/cheyenne-health-care/locations/cheyenne-va-medical-center/
* https://www.va.gov/cheyenne-health-care/locations/fort-collins-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/laramie-va-mobile-clinic/
* https://www.va.gov/cheyenne-health-care/locations/loveland-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/rawlins-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/sidney-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/sterling-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/torrington-va-mobile-clinic/
* https://www.va.gov/cheyenne-health-care/locations/wheatland-va-mobile-clinic/
* https://www.va.gov/erie-health-care/medical-records-office/

## Steps to Recreate
1. Open one of the listed URLs
1. Open axe browser plugin (assumes already installed in Chrome or Firefox)
1. Confirm the `ARIA attributes must conform to valid values` is present one or more times

**Comment by mmiddaugh** [comment]
@1Copenut -
When I looked at the AXE results on this, it looks like it's objecting to the operating status carat. Is that correct?
If so, is the design seen elsewhere on VA.gov? Should it be replaced with an action link or other more accessible design?

**Comment by 1Copenut** [comment]
Good call @mmiddaugh ! The carat is the item in question. I took a quick peek and it's more about there being some HTML that the axe-scanner doesn't like:

```diff
<i
  class="fa fa-chevron-right vads-u-padding-left--0p5"
- "=""
- aria-hidden=" true"
+ aria-hidden="true"
></i>
```
If your team can remove the extra `"="` and the extra space before `true` in the aria-hidden attribute, it should clear this item up.

**Comment by mpelzsherman** [comment]
This has been merged and will go out in today's deploy.

**Comment by mmiddaugh** [comment]
This looks good to me but I'll let @1Copenut confirm before closing. 
I will note that some issues caused by the duplicate accordion labels are still present but I believe that is documented in another issue.

**Comment by mpelzsherman** [comment]
Duplicate accordion labels are due to duplicate data in Drupal. I coded up a fix but was told by @davidconlon to leave the code as is so the duplicates would be surfaced by our axe checks and addressed at the source.

**Comment by 1Copenut** [comment]
This change looks good. I'll move this one to closed.
